src/gui/qgsmaptooldigitizefeature.h

```diff
@@ -17,31 +17,65 @@
 #define QGSMAPTOOLDIGITIZEFEATURE_H
 
 #include "qgsmaptoolcapture.h"
-#include "qgis_app.h"
+#include "qgis_gui.h"
 
 class QgsFeature;
 
-//! This tool digitizes geometry of new point/line/polygon features on already existing vector layers
-class APP_EXPORT QgsMapToolDigitizeFeature : public QgsMapToolCapture
+/**
+ * \ingroup gui
+ * \brief This tool digitizes geometry of new point/line/polygon features on already existing vector layers
+ * Once the map tool is enable, user can digitize the feature geometry
+ * A signal will then be emitted.
+ * \since QGIS 3.10
+ */
+class GUI_EXPORT QgsMapToolDigitizeFeature : public QgsMapToolCapture
```

Comment:
Good job!
Maybe this change deserves a comment on api_break?

Reply:
Is it? because it don't break the API, it adds one more class. Is there a special place to write that API has been augmented?

Reply:
Not really. Additions are not documented.
If someone wants a diff, it should be possible to generate that from the dox (would be a nice little project ;) )